app/scss/indexPage.scss

```diff
@@ -0,0 +1,43 @@
+@import "imports/variables";
+
+* {
+  box-sizing: border-box;
+  body {
+    font-family: 'Asap', sans-serif;
+    .logo {
```

Comment:
The logo style will be the same in every page. 
This could go into styles.scss so that it can be applied to every page.